.eslintrc

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
This file should not be here, as we are going to use Standard

Reply:
ok
i'll fix that